.gitbook/guide/installation.md

```diff
@@ -1,29 +1,52 @@
 # Installation
 
-This guide will explain how to install the `panacead` entrypoints onto your system.
 
 ## Install Go
 
 [Go 1.15+](https://golang.org/doc/install) is required.
 
 ## Install the `panacead`
 
-Install the latest version of Panacea Core.
+If you want to install the `panacead` binary to run your node or communicate with other nodes,
+please clone the `panacea-core` project and build it.
 
 ```bash
 git clone https://github.com/medibloc/panacea-core
 cd panacea-core
-git checkout v2.0.0-alpha.1
+git checkout v2.0.0
```

Comment:
If checkout direct to `v2.0.0` you can also express it. 😄 
```
git clone -b v2.0.0 https://github.com/medibloc/panacea-core
cd panacea-core
```

Reply:
Sure thing. Applied that: 69c3125